kernel/build/config-amd64

```diff
@@ -899,10 +939,8 @@ CONFIG_EFI_PARTITION=y
 # CONFIG_CMDLINE_PARTITION is not set
 # end of Partition Types
 
-CONFIG_BLOCK_COMPAT=y
```

Comment:
```
commit 8f0d196e4dc137470bbd5de98278d941c8002fcb
Author: Lukas Bulwahn <lukas.bulwahn@gmail.com>
Date:   Thu Mar 16 12:16:30 2023 +0100

    block: remove obsolete config BLOCK_COMPAT
    
    Before commit bdc1ddad3e5f ("compat_ioctl: block: move
    blkdev_compat_ioctl() into ioctl.c"), the config BLOCK_COMPAT was used to
    include compat_ioctl.c into the kernel build. With this commit, the code
    is moved into ioctl.c and included with the config COMPAT. So, since then,
    the config BLOCK_COMPAT has no effect and any further purpose.
    
    Remove this obsolete config BLOCK_COMPAT.
    
    Signed-off-by: Lukas Bulwahn <lukas.bulwahn@gmail.com>
    Reviewed-by: Christoph Hellwig <hch@lst.de>
    Acked-by: Arnd Bergmann <arnd@arndb.de>
    Link: https://lore.kernel.org/r/20230316111630.4897-1-lukas.bulwahn@gmail.com
    Signed-off-by: Jens Axboe <axboe@kernel.dk>

```